monasca_rsyslog/client.py

```diff
@@ -28,6 +28,18 @@
 ks_loading.conf.register_conf_options(cfg.CONF, AUTH_CFG_GROUP)
 ks_loading.register_session_conf_options(cfg.CONF, AUTH_CFG_GROUP)
 
+cfg.CONF.register_opts([
+    cfg.StrOpt('region_name',
+               default=None,
+               help='Name of loggin region'),
+    cfg.StrOpt('service_type',
+               default=None,
+               help='Name of logging service type'),
+    cfg.StrOpt('endpoint_type',
+               default=None,
+               help='Name of loggin endpoint type'),
```

Comment:
s/loggin/logging

Include an example? `internal`, `external`

Default to `external`? Most users of this will be sending logs from outside the control plane at a guess.

Reply:
As discussed on slack, I've gone for `public`.